packages/powerzure.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'PowerZure'
+$category = 'Cloud'
+
+$zipUrl = 'https://github.com/hausec/PowerZure/archive/093d4a3267ab514656759d7b2a6a91ac521b449b.zip'
+$zipSha256 = '76e82df57013980cd6f3dd5b125e405e4ab3308368f2372121581a6a1e4a5a22'
+
+VM-Install-Raw-GitHub-Repo $toolName $category $zipUrl $zipSha256
```

Comment:
This is a package that could also benefit from a custom shortcut, as it wants you to import the module

Reply:
@day1player I am not sure I understand what you mean, can you please elaborate?

Reply:
@Ana06 yes, in some of the legacy Commando packages we would create shortcuts that would launch and import the module, then run Get-Help <Main-Function>.. So the LNK file would look something crazy like this:

`cmd /c powershell import-module PowerZure.ps1; Get-Help Get-AzureTarget`

We also set the target icon to be the PowerShell icon, and we set the working directory to the Tools folder directory.